### PR TITLE
demo: fix the scrolling problem when dragging the table

### LINKED ISSUE #45847: table拖拽排序列表时出现拖拽闪动
### Reproduction link

[https://stackblitz.com/edit/react-jzzv12?file=demo.tsx](https://stackblitz.com/edit/react-jzzv12?file=demo.tsx)

### Steps to reproduce

重现步骤：
1. 在table上添加
scroll={{
            x: 'max-content',
}}
2. 鼠标点住一行向下拖不要松

### What is expected?

当拖到最后一行时页面不闪动

### What is actually happening?

拖到最后一行页面闪动

| Environment | Info |
| --- | --- |
| antd | 5.11.1 |
| React | 18.2.01/17.02 |
| System | mac |
| Browser | chrome版本119.0.6045.123 |

### PR DESCRIPTION
**Pull request by linxianxi**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [x] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 💡 Background and solution

fix https://github.com/ant-design/ant-design/issues/45847 https://github.com/ant-design/ant-design/issues/47012

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |    -       |
| 🇨🇳 Chinese |    -       |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: components/table/demo/drag-sorting-handler.tsx [hunk 1]
```diff
@@ -57,13 +59,26 @@ const Row = ({ children, ...props }: RowProps) => {
...
-    transform: CSS.Transform.toString(transform && { ...transform, scaleY: 1 }),
+    transform: CSS.Translate.toString(transform),
     transition,
     ...(isDragging ? { position: 'relative', zIndex: 9999 } : {}),
   };
 
+  // https://github.com/ant-design/ant-design/issues/45847
+  useEffect(() => {
+    trRef.current!.closest('table')!.style.overflow = isDragging ? 'hidden' : '';
+  }, [isDragging]);
```
Comment (afc163) [inline components/table/demo/drag-sorting-handler.tsx:70]: 是不是不要直接修改 dom 元素，而是用 state 去控制？

Comment (linxianxi) [reply components/table/demo/drag-sorting-handler.tsx:70]: 那就要给 table 加一个 className，然后设置它下面的 table 元素 overflow 了。感觉不如这个方便？

Comment (afc163) [reply components/table/demo/drag-sorting-handler.tsx:70]: 目前这个写法依赖 dom 结构，如果 dom 结构变化了可能就不生效了。

Comment (linxianxi) [reply components/table/demo/drag-sorting-handler.tsx:70]: 是的，加 className 的方式也是这样。暂时没找到什么好的办法

**Pull request closed without merging**